deploy-manage/security/secure-your-elastic-cloud-organization.md

```diff
@@ -8,26 +8,32 @@ applies_to:
 
 # Secure your Elastic Cloud organization [ec-securing-considerations]
 
-:::{warning}
-**This page is a work in progress.** 
-:::
+This section describes the settings available for you to secure your {{ecloud}} organization, which is the platform where you can manage your {{ech}} deployments and serverless projects. 
 
+**Managed for you**
 
-## TLS certificate management
+As a managed service, a certain number of [security aspects are handled by Elastic](https://www.elastic.co/cloud/security#details) and don't require specific configuration from you:
 
-TLS certificates apply security controls to network communications. They encrypt data in transit, verify the identity of connecting parties, and help prevent man-in-the-middle attacks.
+- **TLS encrypted communication** is provided in the default configuration. Elasticsearch nodes communicate using TLS.
+- **Encryption at rest**. By default, all of your {{ecloud}} resources are encrypted at rest. Note that you can choose to encrypt your {{ech}} deployments [using your own encryption key](/deploy-manage/security/encrypt-deployment-with-customer-managed-encryption-key.md).
+- **Cluster isolation**. Elasticsearch nodes run in isolated containers, configured according to the principle of least privilege, and with restrictions on system calls and allowed root operations.
 
-For your **{{ech}}** deployments and serverless projects hosted on {{ecloud}}, TLS certificates are managed automatically.
+**Organization-level available security settings**
```

Comment:
```suggestion
**Additional organization-level security settings**
```

maybe slightly clearer

Reply:
if not getting at right message, maybe just remove _**available**_ from original